### DISCUSSION TITLE
How to run at a specific time and every 10 seconds?

### DISCUSSION THREAD

**Discussion by rushkii** [agronholm/apscheduler#605] [Q&A]
Hello, I want to schedule my function every day at 12:00 - 12:20 and run it every 10 seconds, I've search on google search and stackoverflow it doesn't solve my problem, please help me. Thanks.

**Comment by agronholm** [comment]
> Hello, I want to schedule my function every day at 12:00 - 12:20 and run it every 10 seconds, I've search on google search and stackoverflow it doesn't solve my problem, please help me. Thanks.

Please, next time open a discussion instead. This is not a request for enhancement.

So, what have you tried so far? Did you look at the documentation of the `cron` trigger?

**Reply by rushkii** [reply]
I did read the docs but I don't know how to schedule within 12:00 - 12:30 every 10 seconds interval
```
sched.add_job(func, "cron", hour=12, minute="0-30", second=10)
```
this code doesn't solve my problem, I think it just run every a minutes and 10 seconds. Not run every 10 seconds interval within 12:00 - 12:30 range. 🤔

**Reply by agronholm** [reply]
Take a look at this: https://apscheduler.readthedocs.io/en/3.x/modules/triggers/cron.html#expression-types
In particular: `*/a any Fire every a values, starting from the minimum`
So maybe `second="*/10"`?

**Reply by rushkii** [reply]
> Take a look at this: https://apscheduler.readthedocs.io/en/3.x/modules/triggers/cron.html#expression-types In particular: `*/a any Fire every a values, starting from the minimum` So maybe `second="*/10"`?

Thank you so much, I think this is what I looking for.
